src/crimson/os/cyan_store.h

```diff
@@ -21,7 +22,11 @@ class Transaction;
 
 // a just-enough store for reading/writing the superblock
 class CyanStore {
+  constexpr static int MAX_KEYS_PER_OMAP_GET_CALL = 32;
```

Comment:
nit: `int`.

Reply:
Are you saying it should be unsigned?  Or a shorter int type?

Reply:
Apologizes for being not specific. I think we don't need sign here, so an unsigned would fit better. `size_t` instead?

Reply:
probably unsigned would be better? as `size_t` is for size.

Reply:
Switching to unsigned.